doc/sdk-cli-reference.md

```diff
@@ -186,6 +186,20 @@ operator-sdk up local --kubeconfig "mycluster.kubecfg" \
   --operator-flags "--flag1 value1 --flag2=value2"
 ```
 
+The below example will use the default kubeconfig, the default namespace environment var, and pass in flags for the operator.
+To use the operator flags, your operator must know how to handle the option. Below imagine an operator that understands the `resync-interval` flag.
+
+```bash
+operator-sdk up local --operator-flags "--resync-interval 10"
+```
+
+If you are planning on using a different namespace than the default, then you should use the namespace flag to change where the operator is watching for custom resources to be created.
```

Comment:
`should use the namespace flag than the default` to
`should use the --namespace flag than the default namespace`?

Reply:
@fanminshi I'm sorry, but I am not following. 

> should use the namespace flag to change where the operator is watching for custom resources to be created.

Is this the part that you would like changed?

Reply:
sorry for the confustion, I meant 
`If you are planning on using a different namespace than the default, then you should use the namespace flag` ->
`If you are planning on using a different namespace than the default, then you should use the --namespace flag`